scss/detail.scss

```diff
@@ -93,3 +95,63 @@
     margin-right: 12px;
   }
 }
+
+
+@media screen and (max-width: $break-medium) {
+  .book-main {
+    flex-direction: column-reverse;
+  }
+
+  .book-container {
+    flex-direction: column;
+    padding: 38px 18px 43px 18px;
+    margin-bottom: 20px;
+    max-width: 336px;
+  }
+
+  .book-cover {
+    height: 298px;
+    max-width: 212px;
+  }
+
+  .book-image {
+    align-self: center;
+    margin: 0 0 21px 0;
+  }
+
+  .book-title {
+    font-size: 24px;
+    line-height: 38px;
+  }
+
+  .book-gender {
+    font-size: 16px;
+  }
+
+  .book-description {
+    font-size: 14px;
+    line-height: 24px;
+  }
+
+  .book-property {
+    margin-bottom: 18px;
+    width: 100%;
+  }
+
+  .link-back {
+    width: auto;
```

Comment:
This prop is necessary?

Reply:
Yes, since the width must be removed so that it is centered with flexbox